test/unit/Cardano/Wallet/MnemonicSpec.hs

```diff
@@ -0,0 +1,134 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Cardano.Wallet.MnemonicSpec
+    ( spec
+    ) where
+
+
+import Prelude hiding
+    ( length )
```

Comment:
Let's not hide length, but rather use a qualified import on `Data.ByteString` for that :+1:

Reply:
done